components/Notifications/SubscribedAuthors.js

```diff
@@ -65,6 +65,19 @@ const SubscribedAuthors = ({
   data: { authors, myUserSubscriptions, loading, error }
 }) => {
   const [showAll, setShowAll] = useState(false)
+  const [
+    initiallySubscribedAuthorIds,
+    setInitiallySubscribedAuthorIds
+  ] = useState([])
+
+  useEffect(() => {
+    InitializeSubscribedAuthorIds(
+      authors,
+      myUserSubscriptions.subscribedTo.nodes,
+      setInitiallySubscribedAuthorIds
+    )
+  }, [])
```

Comment:
Don't we run into the risk that `authors` and `myUserSubscriptions`are still undefined at this point?